examples/user_guide/Deploy_and_Export.ipynb

```diff
@@ -549,6 +551,53 @@
     "This way we can create a global indicator for the busy state instead of modifying all our callbacks."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### pn.state.schedule\n",
+    "\n",
+    "The `pn.state.schedule` functionality allows scheduling global tasks which will be run in a global context at a periodic interval. Note that the scheduled task **may not** be declared in the application code itself, i.e. if you are serving `panel serve app.py` the callback you are scheduling must not be declared in the `app.py`. It must be defined in an external module or in a `--setup` script declared as part of the `panel serve` invocation.\n",
+    "\n",
+    "Scheduling using `pn.state.schedule` is idempotent, i.e. if a callback has already been scheduled under the same name subsequent calls will have no effect. By default the starting time is immediate but may be overridden with the `at` keyword argument. The period may be declared using the `period` argument or a cron expression (which requires the `croniter` library). Note that the `at` time should be in local time but if a callable is provided it must return a UTC time. If `croniter` is installed a `cron` expression can be provided using the `cron` argument.\n",
+    "\n",
+    "As simple example of a task scheduled at a fixed interval (note that both `async` and regular callbacks are supported):"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import datetime as dt\n",
+    "import asyncio\n",
+    "\n",
+    "async def task():\n",
+    "    print(f'Task executed at: {dt.datetime.now()}')\n",
+    "\n",
+    "pn.state.schedule('task', task, period='1s')\n",
+    "await asyncio.sleep(3)\n",
+    "\n",
+    "pn.state.cancel_scheduled('task')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If you have the `croniter` library installed you may also provide a cron expression, e.g. the following will schedule a task to be repeated at 4:02 am every Monday and Friday:"
```

Comment:
Maybe add a link to what's allowed for a cron expression, either by pointing to that section of croniter or to some cron man page that accurately reflects what's accepted here.